x/stakeibc/keeper/keeper_test.go

```diff
@@ -51,6 +53,16 @@ func TestKeeperTestSuite(t *testing.T) {
 	suite.Run(t, new(KeeperTestSuite))
 }
 
+// Helper function to create an stride epoch tracker that dictates the timeout
+func (s *KeeperTestSuite) CreateStrideEpochForICATimeout(timeoutDuration time.Duration) {
```

Comment:
Awesome! Similar to these setup helpers the actual common tests can be defined here as well.  This is exactly what we need to eventually apply to all the old unit tests in the core core as well! -- all ICAs have a few predictable things not related to their specific logic (like the epoch tracker exists, or the ibc packet sequence increments) so pulling these out into their own helpers is a very nice pattern.  The trickier ones are where you check state before, call the function, then check state after, they way you are doing it here with two helpers is nice but you could have these be even more self contained with the setup/before check happening in a function, and then that same function having a defer to check the after modified state so that you don't need to remember to call both parts.... just an idea to make it even more self contained.

Reply:
Yep discussed this offline, could be a useful refactor in the future but not for now until this has shipped